CHANGES/1564.misc.rst

```diff
@@ -0,0 +1,5 @@
+Fixed unsafe file reading in BufferedInputFile.from_file and 
```

Comment:
What does "unsafe" mean?

--- 

UPD: This comment was the first in the overview of the changes, but after reading all the changes it became clear to me that with these changes you on the contrary make it completely unsafe to use.